integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -124,15 +125,23 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
   this.timeout(60_000); // TODO: Temporarily hardcoded until envs are set up.
   importAppBefore("with-db-flx");
   authenticateUserBefore();
-  openRealmBeforeEach({
-    schema: [FlexiblePersonSchema, DogSchema],
-    sync: {
-      flexible: true,
-    },
-  });
 
   describe("Configuration", function () {
     describe("flexible sync Realm config", function () {
+      it.skip("respects cancelWaitOnNonFatalError", async function () {
+        this.timeout(2_000);
```

Comment:
Is there a reason to shorten the timeout here?

Reply:
The default behaviour is to not cancel on non fatal errors (like a timeout). So in case the test fails the sync client will continue to retry connecting forever. And if we kept the 60 seconds timeout the test will wait the full 60 seconds before failing if the test doesn't pass.

Reply:
Great reason then 👍🏼